contributor/Development_guide.md

```diff
@@ -0,0 +1,41 @@
+# Development guide
+
+## Prerequisites
+The OpenLab CI is built based on `NodePool` and `Zuul` tools of the OpenStack infrastructure. The CI Jobs definition uses Ansible playbooks. Learn more about the tools first:
+* Zuul documentation is hosted on [Zuul](https://zuul-ci.org)
+* NodePool documentation is hosted on [NodePool](https://zuul-ci.org/docs/nodepool/)
+* Ansible documentation is hosted on [Ansible Docs](https://docs.ansible.com/)
+
+### Testing workflow 
+   <img src="../images/testing_workflow.png" width="80%"/>
+
+### OpenLab CI arch
+   <img src="../images/OpenLab_CI_arch.png" width="80%"/>
+ 
+  
+## Working on test request with OpenLab CI
+The accepted test requests can be found on [TODO List](https://github.com/orgs/theopenlab/projects/1#column-1860008), developers can get start with following steps:
+1. Pick one test request you wish to work on, move it to [IN PROGRESS list](https://github.com/orgs/theopenlab/projects/1#column-1860011), edit it to add the development plan into comments
+2. The target project of the request should install the **OpenLab-CI** [github APP](https://github.com/apps/theopenlab-ci), so that OpenLab CI system and target project can communicate with each other. See the details on [connecting guideline](./Connect_to_OpenLab.md)
+3. Fork your own copy of the CI Job repository [openlab-zuul-jobs](https://github.com/theopenlab/openlab-zuul-jobs) to your account, develop the test job and submit a PR(Pull Request) to [openlab-zuul-jobs](https://github.com/theopenlab/openlab-zuul-jobs). In order to manage the test request well, please please add change details and link the test request number in PR's commit message, the format like:
+   
+   `Related-Bug: theopenlab/openlab#number` 
+4. OpenLab development team @theopenlab/dev will review the PR and give some suggestion, development core team @theopenlab/dev-core have permission to merge it if there is no issue, at the meanwhile, you should contact @theopenlab/dev-core to add the target project into **main.yaml** in **zuul** node if it is not in project list of zuul
+5. When your PR is merged into master of [openlab-zuul-jobs](https://github.com/theopenlab/openlab-zuul-jobs), trigger to test the integration job by [trigger comments](./Supported_trigger_comments.md), see the [build status](http://status.openlabtesting.org/t/openlab/builds.html), click `logs` to get the log details if the job fail. Maybe you can find some problems of job, feel free to submit new pull request to [openlab-zuul-jobs](https://github.com/theopenlab/openlab-zuul-jobs/pulls) to fix job issue, link the issue number of related test request in PR's commit message as we do on step3  
```

Comment:
We apply new release portal of OpenLab CI after latest monthly upgrading, so please update this describe to match new links, and `logs` have been move into STATUS link.